extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -337,11 +341,6 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
   };
 
   private hasBodyChanged = (msgBody: string) => {
-    if (this.lastDraftBody === undefined) {
-      // first check
-      this.lastDraftBody = msgBody;
-      return false;
-    }
```

Comment:
This check was used for existing drafts, after it's removing existing drafts are re-saved even without changes:
1) create and save draft
2) go to Drafts folder
3) open this draft
4) wait for 3 seconds (without any changes)
5) draft is saved and `last edit` time is changed

Existing draft shouldn't be re-saved when no changes applied.
Probably it's possible to create such ui test?

Reply:
I see. I thought adding this line of code would also cover this case but I added in wrong way. Just fixed it.
Let me add UI test.
https://github.com/FlowCrypt/flowcrypt-browser/blob/8da995814e20e971f61b211155b5dcf43ddf8a17/extension/chrome/elements/compose-modules/compose-input-module.ts#L47